synapse/lib/auth.py

```diff
@@ -124,6 +124,8 @@ async def __anit__(self, slab, dbname, pref='', nexsroot=None, seed=None, maxuse
             slab (s_lmdb.Slab): The slab to use for persistent storage for auth
             dbname (str): The name of the db to use in the slab
         '''
+        if policy:
+            s_schemas.reqValidPasswdPolicy(policy)
```

Comment:
Doesn't this happen already at startup?

Reply:
With moving the schema from the confdef we would not longer be checking it at config time.